projects/packages/account-protection/src/class-validation-service.php

```diff
@@ -273,4 +325,15 @@ public function is_recent_password( int $user_id, string $password ): bool {
 
 		return false;
 	}
+
+	/**
+	 * Get the old user data.
+	 *
+	 * @param int $user_id The user ID.
+	 *
+	 * @return \WP_User|false The old user data, or false if the user does not exist.
+	 */
+	public function get_old_user_data( $user_id ) {
+		return get_userdata( $user_id );
+	}
```

Comment:
What is the purpose of adding this method to wrap `get_userdata`, instead of using the core method in implementation directly?

Reply:
I believe it was only added for testing purposes because you can't or couldn't easily mock core functions.